roofit/hs3/test/histfactory.py

```diff
@@ -5,7 +5,7 @@
 msg.setGlobalKillBelow(ROOT.RooFit.WARNING)
```

Comment:
Please rename the `histfactory.py` file to `test_hs3_histfactory_json.py`, which is less ambiguous, follows the convention for Python test files and is the name expected by the `CMakeLists.txt` in the same directory (the filename inconsistency was the reason why the test was not run correctly by ctest).